x/tokenomics/keeper/token_logic_modules.go

```diff
@@ -245,23 +274,24 @@ func (k Keeper) ProcessTokenLogicModules(
 func (k Keeper) TokenLogicModuleRelayBurnEqualsMint(
 	ctx context.Context,
 	service *sharedtypes.Service,
+	_ *sessiontypes.SessionHeader,
 	application *apptypes.Application,
 	supplier *sharedtypes.Supplier,
 	settlementCoin cosmostypes.Coin,
 	relayMiningDifficulty *tokenomictypes.RelayMiningDifficulty,
 ) error {
 	logger := k.Logger().With("method", "TokenLogicModuleRelayBurnEqualsMint")
 
+	// DEV_NOTE: We are doing a mint & burn + transfer, instead of a simple transfer
+	// of funds from the supplier to the application in order to enable second
+	// order economic effects with more optionality. This could include funds
+	// going to pnf, delegators, enabling bonuses/rebates, etc...
+
 	ownerAddr, err := cosmostypes.AccAddressFromBech32(supplier.OwnerAddress)
 	if err != nil {
 		return err
 	}
 
-	// NB: We are doing a mint & burn + transfer, instead of a simple transfer
-	// of funds from the supplier to the application in order to enable second
-	// order economic effects with more optionality. This could include funds
-	// going to pnf, delegators, enabling bonuses/rebates, etc...
-
 	// Mint new uPOKT to the supplier module account.
 	// These funds will be transferred to the supplier below.
```

Comment:
```suggestion
	// Mint new uPOKT to the supplier module account.
	// These funds will be transferred to the supplier and revenue share-holders below.
```

Reply:
I'm not seeing where the supplier balance is modiefied; i.e., where the supplier is paid. :thinking:

It looks like we're only doing the following:
1. Minting to the supplier module account
2. Distributing newly minted tokens (from the supplier module account) to rev-share-holders

Am I missing something or do we lacking test coverage around this?

Reply:
This was a new feature implemented by @red-0ne: https://dev.poktroll.com/operate/configs/supplier_staking_config#default_rev_share_percent

THEORETICALLY, the `operator` and `owner` don't have to receive any of the rewards from claims & proofs.

In practice, they almost always are part of the list.